style.css

```diff
@@ -0,0 +1,223 @@
+    border-radius: 5px;
+    padding: .5em;
+}
+
+.quote {
+    font-family: 'Rokkitt', serif;
+    font-style: italic;
+}
+
+.quote blockquote {
+    text-align: left;
+}
+
+.quote p {
+    text-align: right;
+}
+
+#b-quote-1 {
+    grid-row: 3;
+    /*grid-column: 2;*/
+    padding: 50px 110px;
+    border-bottom: solid lightcoral 1px;
+}
+
+.features {
+    grid-row: 4;
+    /*grid-column: 2;*/
+    background-color: greenyellow;
+    border-bottom: solid lightcoral 1px;
+    /*text-align: left;*/
+    /*display: grid;*/
+    /*grid-template-columns: repeat(3, 1fr);*/
+    /*grid-template-rows: 3em 60px 60px;*/
+}
+.features > h2 {
+    text-align: center;
+}
+.features img {
+    float: left;
+}
+
+.features ul{
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: center;
+    /*align-content: center;*/
+
+}
+/*.f1 strong {*/
+    /*display: inline;*/
+    /*height: 1em;*/
+/*}*/
+
+/*.f1 p {*/
+    /*margin-top: 0;*/
+    /*display: inline;*/
+    /*text-align: left;*/
+/*}*/
+
+
+/*.f1 {*/
+    /*!*vertical-align: middle;*!*/
+    /*display: flex;*/
+    /*border: 1px solid black;*/
+    /*width: 200px;*/
+    /*flex-direction: row;*/
+    /*flex-wrap: wrap;*/
+
+/*}*/
+/*.fi li{*/
+    /*display: flex;*/
+    /*border: gray 1px solid;*/
+/*}*/
+
+#b-quote-2{
+    grid-row: 5;
+    /*grid-column: 2;*/
+    padding: 50px 110px;
+    border-bottom: solid lightcoral 1px;
+}
+
+.team > h2 {
+    text-align: center;
+}
+
+.team {
+    background-color: orangered;
+    grid-row: 6;
+    /*grid-column: 2;*/
+}
+
+/*.team-members li img, span {*/
+    /*display: flex;*/
+    /*width: 100px;*/
+    /*flex-direction: row;*/
+    /*align-items: flex-start;*/
+    /*justify-content: space-evenly;*/
+    /*border-bottom: solid lightcoral 1px;*/
+/*}*/
+
+.team-members li img{
+    border-radius: 50%;
+    -webkit-transition: -webkit-transform .8s ease-in-out;
+    transition:         transform .8s ease-in-out;
+}
+
+.team-members li img:hover {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+}
+.team-members ul {
+    display: flex;
+    flex-flow: row wrap;
+    width: 100px;
+    /*align-content: space-around;*/
+    justify-content: space-evenly;
+    border-bottom: solid lightcoral 1px;
+}
+/*.team-members li {*/
+    /*background-color: cornflowerblue;*/
+    /*display: flex;*/
+    /*width: 75px;*/
+    /*border: 1px solid black;*/
+    /*justify-content: space-evenly;*/
+/*}*/
+
+.sponsors {
+    grid-row: 7;
+    /*grid-column: 2;*/
+    border-bottom: solid lightcoral 1px;
+}
+
+.sponsors img[src]{
+    padding: 5px;
+}
+
+.sponsors ul {
+    display: flex;
+    background: yellow;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    padding: 20px;
+
+}
+
+
+.footer {
+    grid-row: 8;
+    grid-column: 1 / -1;
+    display: flex;
+    padding-left: 200px;
+    align-content: flex-start;
+}
+.footer h3{
+    text-align: left;
+}
+.footer div ul {
+    display: inline;
+    text-align: left;
```

Comment:
I feel so ridiculous for not thinking to just use text-align on the footer lists. Instead I did some really hacky stuff with flexbox and negative margins lol. Good work!!